server/blob/blob-deduplicating/src/test/scala/org/apache/james/server/blob/deduplication/State.scala

```diff
@@ -0,0 +1,46 @@
+/***************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.server.blob.deduplication
+
+/**
+ * Used to iteratively build a StabilizedState
+ */
+case class State(references: Map[Generation, Seq[Reference]], deletions: Map[Generation, Seq[Deletion]]) {
```

Comment:
why only this class have been put in an other file?

Reply:
It is neither related to the GC, nor to its properties.

Reply:
Why is this in `test` package?

Reply:
Because it is only useful here